agent-zero/skills/blender-3d-studio/scripts/start-mcp-server.sh

```
@@ -0,0 +1,883 @@
+# Make server script executable
+chmod +x "$MCP_SERVER_SCRIPT"
+
+# Set environment variables for the server
+export MCP_HOST="$HOST"
+export MCP_PORT="$SERVER_PORT"
+export MCP_ENABLE_API="$ENABLE_API"
+export MCP_ENABLE_RENDER_QUEUE="$ENABLE_RENDER_QUEUE"
+export MCP_MAX_JOBS="$MAX_CONCURRENT_JOBS"
+export MCP_CACHE_DIR="$CACHE_DIR"
+export MCP_LOG_LEVEL="$LOG_LEVEL"
+export BLENDER_PATH="$blender_path"
```

Shellcheck (0.11.0)

[warning] blender_path is referenced but not assigned (did you mean 'BLENDER_PATH'?).

(SC2154)